docs/source/guides/integrations/basic_aim_mlflow_logs_conversion.md

```diff
@@ -0,0 +1,64 @@
+### Show TensorFlow events in Aim
+
+Aim gives your possibility to convert [MLflow](https://mlflow.org/) runs into native format and show them directly
+inside the Aim interface.
+
+Before showing your MLlfow runs in Aim, they need to pass conversion process where your metrics, tags, parameters, run
+description/notes and *some* artifacts will be transferred into Aim storage and shown inside Aim web interface.
+
+Please note that as for now, only the artifacts having the following file extensions will be transferred into Aim
+storage!
+
+* Images: `(
+  'jpg',
+  'bmp',
+  'jpeg',
+  'png',
+  'gif',
+  'svg'
+  )`
+
+* Texts: `(
+  'txt',
+  'log',
+  'py',
+  'js',
+  'yaml',
+  'yml',
+  'json',
+  'csv',
+  'tsv',
+  'md',
+  'rst',
+  'jsonnet'
+  )`
+
+* Sounds/Audios: `(
+  'flac',
+  'mp3',
+  'wav'
+  )`
+
+To convert MLflow runs, `aim convert mlflow` command must be run on your log directory:
+
+```commandline
+$ aim init
+$ aim convert mlflow --tracking_uri 'file:///Users/aim_user/mlruns'
+```
+
+The conversion process will iterate over all your Experiments and create a distinct run for each run inside the
+experiment. If you want to process only a single experiment, you can provide the experiment id to the conversion
+command:
+
+```commandline
```

Comment:
worth adding a section for converting remote logs as well? also I remember mlflow provides env. variables for authentication and stuff like that in case of cloud storage. it would be good to show an example where these env. vars are set and remote artifact storage is used.

Reply:
added some entry taken from MLFlow docs